engine/src/main/java/org/hibernate/validator/internal/engine/messageinterpolation/el/SimpleELContext.java

```diff
@@ -6,74 +6,52 @@
  */
 package org.hibernate.validator.internal.engine.messageinterpolation.el;
 
-import java.lang.reflect.Method;
-
 import javax.el.ArrayELResolver;
 import javax.el.BeanELResolver;
 import javax.el.CompositeELResolver;
-import javax.el.ELContext;
 import javax.el.ELResolver;
 import javax.el.ExpressionFactory;
 import javax.el.ListELResolver;
 import javax.el.MapELResolver;
 import javax.el.ResourceBundleELResolver;
-import javax.el.ValueExpression;
-import javax.el.VariableMapper;
+import javax.el.StandardELContext;
+import javax.el.StaticFieldELResolver;
 
 /**
  * @author Hardy Ferentschik
+ * @author Guillaume Smet
  */
-public class SimpleELContext extends ELContext {
+public class SimpleELContext extends StandardELContext {
```

Comment:
> So if we want to be sure it's used, we need to put it in the context.

Yes, but couldn't it be done like this:

    ELContext ctx  = new StandardELContext( factory );
    ctx.putContext( ExpressionFactory.clss, factory );

> we instantiate new resolvers for each interpolation

Mh, that sounds bad. Where is it happening?